docs/ja/docs/advanced/templates.md

```diff
@@ -0,0 +1,77 @@
+# テンプレート
+
+お好きなテンプレートエンジンを**FastAPI**と使うことができます。
+
+よく使われるものは Flask などで使われているJinja2です。
+
+Starletteにより提供されている簡単に設定できるユーティリティーがあり、**FastAPI**アプリケーションで直接使用することができます。
+
+## 依存関係のインストール
+
+`jinja2`のインストール:
+
+<div class="termy">
+
+```console
+$ pip install jinja2
+
+---> 100%
+```
+
+</div>
+
+## `Jinja2Templates`の使用
+
+* `Jinja2Templates`をインポートします。
+* あとで再利用可能な`templates`オブジェクトを作ります。
+* テンプレートを返す*パスオペレーション*内で`Request`パラメータを宣言します。
+* 作成した`templates`を使って`TemplateResponse`をレンダリングし、Jinja2の"context"のキーと値のペアの一つとして`request`を渡します。
+
+```Python hl_lines="4  11  15-16"
+{!../../../docs_src/templates/tutorial001.py!}
+```
+
+!!! note
+`request`をJinja2のコンテキストのキーと値ののペアとして渡さなければいけないことに注意してください。そのため、*パスオペレーション*の中でもそれを宣言しなければなりません。
```

Comment:
```suggestion
    `request`をJinja2のコンテキストのキーと値ののペアとして渡さなければいけないことに注意してください。そのため、*パスオペレーション*の中でもそれを宣言しなければなりません。
```

A following sentence in a tips or note block needs indent like English. If no indent, cause broken css style and not was surrounded block.

https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/advanced/templates.md?plain=1#L35